doc/frame-messenger.md

```diff
@@ -8,10 +8,13 @@ Tools like browser extensions and testing environments often have different chan
 axe.frameMessenger({
   // Called to initialize message handling
   open(topicHandler) {
+    // Map data from the bridge to
+    function subscriber(frameWin, data, response) {
+      // Data serializations / validation / etc. here
```

Comment:
nit: this would probably be where an implementer would be *de*serializing data, not serializing it